site/gatsby-site/src/graphql/schema.js

```diff
@@ -0,0 +1,4282 @@
+export const typeDefs = `
```

Comment:
This file looks like it's going to be fun to maintain, but hopefully easier (with better errors for development) than the current setup in the long run if we can break it down and compose the pieces.